### ISSUE TITLE
Does ron support JSON schemas for enabling intellisence in configs?

### ISSUE THREAD

**Issue by EmilySeville7cfg** [ron-rs/ron#470]
I wanna have something like this in .ron files:

![image](https://github.com/ron-rs/ron/assets/42812113/288c5640-5734-47fb-96ff-6eabe7585147)

**Comment by juntyr** [comment]
There is a VS-code extension for RON, https://github.com/a5huynh/vscode-ron, does it provide what you’re looking for? Otherwise I unfortunately don’t know either but would be happy to learn if some exists and we can promote it through our README

**Comment by EmilySeville7cfg** [comment]
No, it doesn't provide all features I need: it misses syntax checking. There is just syntax highlighting available here.

**Comment by juliancoffee** [comment]
https://github.com/jasonjmcghee/ron-lsp

This exists, fyi.
